### PR TITLE
Workshop feedback changes round 2

### PR DESCRIPTION
**Pull request by Reb-B**
The changes from the second round of feedback are implemented here. The new lab names are just suggestions.

### CODE REVIEW

File: content/02_installation/_index.md [hunk 1]
```diff
@@ -20,8 +20,8 @@ if these are in place, you can now begin the installation process.
 
 Getting started with [superwerker] is simple: You only need to deploy a single AWS CloudFormation template into an existing AWS account.
 
-1. Sign into your AWS account if you are not already logged in.
-1. Select to AWS region in which you want to deploy superwerker.
+1. Sign into your AWS account with you root user email and password if you are not already logged in.
```
Comment (sbstjn) [inline content/02_installation/_index.md:23]: your

File: content/04_costs/_index.md [hunk 2]
```diff
@@ -23,6 +23,6 @@ The goal of [superwerker] is to provide you with a set of well-configured AWS se
 
 > As a rule of thumb: The costs for these services will grow with the number of AWS accounts and the number of workloads (EC2 instances, databases, Lambda functions, …) you use.
 
-Given the benefits of a multi-account AWS environment, this is a justifiable invest.
+Based on experience from multiple customer engagements, moving from a single AWS account setup to a multi-account setup later, when some applications and teams are already working on AWS, results in a higher investment compared to the monthly cost of starting out with a fresh install of superwerker.
```
Comment (sbstjn) [inline content/04_costs/_index.md:26]: Can we come up with a reason? Like "_doing things right in the first place is cheaper on a long perspective, than cleaning up a mess after a while_"

File: content/05_labs/01_sso/_index.md [hunk 3]
```diff
@@ -1,20 +1,20 @@
 ---
-title: "Single Sign-On"
+title: "Signing in securely"
```
Comment (sbstjn) [inline content/05_labs/01_sso/_index.md:2]: What about "Secured Access" or "Secure Access to AWS" or something like that?

File: content/05_labs/02_org_setup/_index.md [hunk 4]
```diff
@@ -1,18 +1,18 @@
 ---
-title: "AWS Organization"
+title: "Organizing accounts"
```
Comment (sbstjn) [inline content/05_labs/02_org_setup/_index.md:2]: Don't write headlines with present participle! (I just made up that rule)

**Review by sbstjn — CHANGES_REQUESTED** [review]

File: content/05_labs/01_sso/_index.md (hunk 3 above)
Comment (Reb-B) [reply content/05_labs/01_sso/_index.md]: I went with "Secure Access to AWS"

File: content/05_labs/02_org_setup/_index.md (hunk 4 above)
Comment (Reb-B) [reply content/05_labs/02_org_setup/_index.md]: 'Present' is hands-down the best participle but i have changed it to "Account Organization" for now.

**Pull request merged**